### PR TITLE
Signup thank you content update

### PR DESCRIPTION
**Pull request by EphraimAssi**
# Description

[Update content on sign up Thank You page as per figma](https://trello.com/c/iu8jyB0p/312-update-content-on-sign-up-thank-you-page-as-per-figma)

I updated the sign-up page content.
image
image

I also updated the thank you page.
image

I also did some small changes to the select field to make the icon always stay at the same place.
image
image

## Acceptance Criteria

The thank you and sign-up page should be like the same content as figma.

## Test Instructions

1. Check the sing-up page and make sure it's the same as figma
2. Check the errors on the sign-up page also
3. Check the thank you page
4. Check the sign-up page dropdown in different screen widths to see if the icon doesn't move around

## Help Requested

There is now a resend email link, I put it, but I'm not sure how to implement it yet, so it's there but doesn't work.

## Checklist

- [ ] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
   "active": "Active",
   "filterBy": "Filter by",
   "skipToMainContentBtn": "Skip to main content",
-  "signupTitle": "Become a participant!",
+  "signupTitle": "Sign up become a participant!",
```
Comment (var-kyle) [inline public/locales/en/common.json:86]: Small typo, it should be `Sign up to become a participant!`

File: public/locales/en/common.json [hunk 2]
```diff
@@ -177,11 +180,12 @@
   "privacyLinkText": "Review the privacy policy",
   "becomeAParticipantDescription": "You’re invited to test our experimental ideas and participate in research interviews to make Service Canada better for everyone. Every bit of feedback helps us make sure we’re making our services simple and easy to use.",
   "thankYou": "Thank you!",
-  "confirmationP1": "We’ve added you to our list and sent you a confirmation email from",
+  "confirmationP1": "We have sent a confirmation email to <<youremail@email.com>> from",
```
Comment (var-kyle) [inline public/locales/en/common.json:183]: Just a question, but are we expected to replace `<<youremail@email.com>>` with the actual email address from the user?

File: public/locales/fr/common.json [hunk 3]
```diff
@@ -182,6 +185,7 @@
   "confirmationP2": "Nous vous inviterons à prendre part aux expériences et à d’autres activités de recherche au fur et à mesure qu’elles se présenteront afin d’améliorer Service Canada pour tous.",
   "confirmationP3": "N’oubliez pas que vous pouvez choisir le temps que vous y consacrez et que chaque petit geste compte.",
   "confirmationP4": "Nous sommes véritablement heureux de vous avoir dans l’équipe!",
+  "resendEmail": "Renvoyer le courriel",
```
Comment (var-kyle) [inline public/locales/fr/common.json:188]: The word doc with initial translations has this as `Envoyer de nouveau un courriel de vérification `

File: public/locales/fr/common.json [hunk 4]
```diff
@@ -192,5 +196,6 @@
   "scLabsThankYou": "Laboratoires de Service Canada - Merci",
   "scLabsExperiments": "Laboratoires de Service Canada - Expérimentations",
   "scLabsConfirmation": "Laboratoires de Service Canada - Confirmation",
-  "scLabsAbout": "Laboratoires de Service Canada - À propos"
+  "scLabsAbout": "Laboratoires de Service Canada - À propos",
```
Comment (var-kyle) [inline public/locales/fr/common.json:199]: This should be `À propos des laboratoires`

**Review by var-kyle — CHANGES_REQUESTED** [review]
Looks mostly good, just a few typos to fix.

**Review by will0684 — COMMENTED** [review]
Any chance [this card](https://trello.com/c/RmDmXJqO/316-correct-the-signup-thank-you-page-to-indicate-the-right-email-from-which-users-will-receive-their-confirmation-email) could be addressed as part of this PR?

File: public/locales/en/common.json (hunk 2 above)
Comment (EphraimAssi) [reply public/locales/en/common.json:183]: I'm not sure actually.

**Comment by EphraimAssi** [comment]
I also added the new email for the thank you page.

File: pages/thankyou.js [hunk 5]
```diff
@@ -46,9 +47,9 @@ export default function Confirmation(props) {
               {t("confirmationP1")}&nbsp;
               <a
                 className="underline hover:text-canada-footer-hover-font-blue text-canada-footer-font"
-                href={`mailto: ${process.env.NEXT_PUBLIC_NOTIFY_REPORT_A_PROBLEM_EMAIL}`}
+                href={`mailto: ${process.env.NEXT_PUBLIC_THANK_YOU_EMAIL}`}
               >
-                {t("confirmationEmail")}
+                alpha@notification.canada.ca
               </a>
```
Comment (var-kyle) [inline pages/thankyou.js:53]: If these are the same email, the text doesn't need to be hard coded, you can get it from the environment as well.

**Review by var-kyle — DISMISSED** [review]

**Review by will0684 — APPROVED** [review]

**Comment by will0684** [comment]
Looking good thanks Ephraim

**Review by will0684 — APPROVED** [review]
Thank you!

**Pull request merged**